tests/units/array/test_array_from_to_csv.py

```diff
@@ -0,0 +1,153 @@
+import os
+from typing import Optional
+
+import pytest
+
+from docarray import BaseDocument, DocumentArray
+from docarray.array.array.io import (
+    _dict_to_access_paths,
+    _update_nested_dicts,
+    is_access_path_valid,
+)
+from docarray.documents import Image
+from tests import TOYDATA_DIR
+
+
+class MyDoc(BaseDocument):
+    count: Optional[int]
+    text: str
+
+
+class MyDocNested(MyDoc):
+    image: Image
+    image2: Image
```

Comment:
either create fixture or copy paste the doc in the tests.

otherwise one test could modify one class and it could impact another test